adoc/SAP-EIC-Main.adoc

```diff
@@ -20,8 +20,6 @@ It will guide you through the steps of:
 * Deploying mandatory components for {eic}
 // * Deploying {eic} into your {rke}
 
-include::SAP-EIC-General.adoc[tags=disclaimer-EIC-sizing;disclaimer-annotation]
```

Comment:
Has this include been deleted by purpose from the MAIN file adoc/SAP-EIC-Main.adoc ?? I see it is still used in the MAIN file adoc/SAP-EIC-Harvester-main.adoc

Reply:
Spot on! I deleted the whole line instead of the _**disclaimer-annotation**_ .